### PR TITLE
Reset builder to use plasmapy's doc builder image

### LINKED ISSUE #25: CircleCI integration
Implement CircleCI for continuous integration, similar to PlasmaPy.

### LINKED ISSUE #49: setup Giles

### LINKED ISSUE #55: change readthedocs template
Change template to be consistent with PlasmaPy?

### PR DESCRIPTION
**Pull request by StanczakDominik**
Closes #25. Closes #55. Closes #49 (Giles sort of works intermittenly so I'm not 100% sure but it should work).

### CODE REVIEW

**Comment by StanczakDominik** [comment]
> 100% coverage

**Comment by StanczakDominik** [comment]
I might actually give up on doing this from scratch and just copy over Cadair's changes from PlasmaPy v0.1...

**Comment by StanczakDominik** [comment]
All right, this is just about done! I ended up merging some of the retemplating work I did earlier and current PlasmaPy repo state, so this could probably be cleaner... but it works, and that's probably the important part.

Oh look, and there's a Giles doc build! Neat.

**Comment by StanczakDominik** [comment]
Oh yeah, @lemmatum, once you merge this, it would probably make sense to merge master into the other three PRs. That, or we could rebase them - but mayyyybe let's not do that. :smile:

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/PlasmaPy/SpectroscoPyx/badge.svg?branch=master)](https://coveralls.io/github/PlasmaPy/SpectroscoPyx?branch=master)
 [![Matrix](https://matrix.to/img/matrix-badge.svg)](https://riot.im/app/#/room/#plasmapy:matrix.org)
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PlasmaPy/Lobby)
+[![Powered by Astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat)](http://www.astropy.org)
```
Comment (lemmatum) [inline README.md]: :smile:

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,4 @@
 [submodule "astropy_helpers"]
-	path = astropy_helpers
 	url = https://github.com/astropy/astropy-helpers.git
+	path = astropy_helpers
+	branch = refs/heads/v3.0.1
```
Comment (lemmatum) [inline .gitmodules]: Will this have to be adjusted every time astropy_helpers gets updated?

File: docs/conf.py [hunk 3]
```diff
@@ -1,181 +1,180 @@
...
+master_doc = 'index'
 
-# This does not *have* to match the package name, but typically does
-project = setup_cfg['package_name']
-author = setup_cfg['author']
-copyright = '{0}, {1}'.format(
-    datetime.datetime.now().year, setup_cfg['author'])
+# General information about the project.
+project = 'SpectroscoPyx'
+copyright = '2015-2018, SpectroscoPyx Community'
```
Comment (lemmatum) [inline docs/conf.py]: 2017-2018

File: setup.py [hunk 4]
```diff
@@ -66,7 +75,7 @@
 builtins._ASTROPY_PACKAGE_NAME_ = PACKAGENAME
 
 # VERSION should be PEP440 compatible (http://www.python.org/dev/peps/pep-0440)
-VERSION = metadata.get('version', '0.1.dev1')
+VERSION = metadata.get('version', '0.0.dev0')
```
Comment (lemmatum) [inline setup.py]: This doesn't match the version in other places, e.g. 0.1dev3 under setup.cfg - is that a problem?

File: .gitmodules (hunk 2 above)
Comment (StanczakDominik) [reply .gitmodules]: Yeah, but you're supposed to only really handle that via git submodule commands.

File: setup.py (hunk 4 above)
Comment (StanczakDominik) [reply setup.py]: It doesn't because I was doing this in a rush and completely forgot about other places that have versions. :sweat_smile:

**Comment by lemmatum** [comment]
Merged, thank you @StanczakDominik !

**Pull request merged**